Source/ACE.Entity/AceCharacter.cs

```diff
@@ -0,0 +1,16 @@
+﻿using ACE.Common;
+using MySql.Data.MySqlClient;
+
+namespace ACE.Entity
+{
+    [DbTable("vw_ace_character")]
+    [DbGetList("vw_ace_character", 42, "guid")]
+    public class AceCharacter
```

Comment:
It is the start of the orm work.   I knew it was complete.   I can take it out, but it is the next thing I am working on.   Let me know if it needs to come out.   I started on the wrong branch.